api/src/main/java/org/openmrs/module/fhir2/api/translators/VisitReferenceTranslator.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators;
+
+import org.hl7.fhir.r4.model.Reference;
+import org.openmrs.Encounter;
+import org.openmrs.Visit;
+
+public interface VisitReferenceTranslator extends OpenmrsFhirTranslator<Visit, Reference> {
```

Comment:
So, in the interim between when you started working on this and now, we added a `VisitReferenceTranslatorImpl` class, which can be gotten by requesting an instance of `EncounterReferenceTranslator<Visit>`. It would probably be a good idea to use that.